.env.example

```
@@ -42,6 +42,13 @@ LOG_PRETTY=false
 # Prometheus /metrics endpoint 주소 (빈 문자열이면 비활성화)
 METRICS_ADDR=:9090
 
+# Graceful shutdown timeout 설정 (이슈 #272)
+# - SHUTDOWN_TIMEOUT: stages.Stop / worker.Stop 전체 timeout (default 30s)
+#   claudegen LLM 추출기 활성 시 sonnet latency p95=110s 라 120s 권장.
+# - CLAUDE_CODE_SHUTDOWN_TIMEOUT: claudegen 컨테이너 cleanup (docker rm -f) timeout (default 10s)
+SHUTDOWN_TIMEOUT=30s
+CLAUDE_CODE_SHUTDOWN_TIMEOUT=10s
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CLAUDE_CODE_SHUTDOWN_TIMEOUT key should go before the SHUTDOWN_TIMEOUT key

(UnorderedKey)